kolibri/core/assets/src/views/k-radio-button/index.vue

```diff
@@ -11,6 +11,7 @@
         <input
           ref="kRadioInput"
           type="radio"
+          :autofocus="autofocus"
```

Comment:
Move below `:disabled=`

Reply:
... why?

Reply:
Personally I like to organize my props by type and match the order of the props object if possible.